$NetBSD: patch-ac,v 1.3 2009/09/19 00:05:03 minskim Exp $

CVS Revision 1.13
Fix: awstats emits ton of warnings with new geoipfree - ID: 2794728

--- wwwroot/cgi-bin/plugins/geoipfree.pm.orig	2008-11-15 08:03:42.000000000 -0800
+++ wwwroot/cgi-bin/plugins/geoipfree.pm
@@ -51,7 +51,7 @@ sub Init_geoipfree {
 	# ENTER HERE CODE TO DO INIT PLUGIN ACTIONS
 	debug(" Plugin geoipfree: InitParams=$InitParams",1);
 	%TmpDomainLookup=();
-	$gi = Geo::IPfree::new();
+	$gi = Geo::IPfree->new();
 #	$gi->Faster; 	# Do not enable Faster as the Memoize module is rarely available
 	# ----->
 
